en/docs/references/concepts/authentication/dcr.md

```diff
@@ -2,13 +2,12 @@
 
 This extension provides a mechanism to register clients with the authorization server dynamically or programmatically. 
  
-
-## How it works 
+## How does it work? 
 
 To register a new client at the authorization server:
 
 1. Client discovers the client registration endpoint. The registration endpoint URL is discoverable via WebFinger.
-   For more information on discovering the client registration endpoint, see [OpenID Connect Discovery](../login/discovery.md).
+   For more information on discovering the client registration endpoint, see [OpenID Connect Discovery](discovery.md).
```

Comment:
Did we have to mention the .md part to make it work?

Reply:
yes it is needed